sparkler-api/src/test/resources/user-agents.txt

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# User agents to be used
```

Comment:
Anytime prof. Very glad that I got your review. Let me explain.

I think we are saying the same thing. I suspect not all nutch protocol-* plugins rotate user agents. I just checked with [lib-htmlunit](https://github.com/apache/nutch/blob/master/src/plugin/lib-htmlunit/src/java/org/apache/nutch/protocol/htmlunit/HtmlUnitWebDriver.java)

> In addition I guess I don't understand why rotating agents (including the user-agents.txt file is part of this PR)

Because 
1.  the test for default fetcher that is part of core system is written using the default file from [conf/user-agents.txt](https://github.com/USCDataScience/sparkler/blob/master/conf/user-agents.txt)

2. Another plugin based on Jbrowser doesnt have user agent rotation functionality. So it didnt have this file. 

3. This plugin rotates agent so it needed a test. In addition, I thought it will be a good practice to keep a separate file for tests so I introduced this file. I agree I should have introduced this in default fetcher described in 1, its never too late to fix the mistake.